### PR TITLE
Fix disconnect - Update Menu Bar

### PR DESCRIPTION
**Pull request by makeSmartio**
I am new to Pull Requests - apologies if I did this wrong. This line of code updates the display after the AirPods are disconnected.

### CODE REVIEW

File: AirpodsPro Battery/AirpodsPro Battery/ViewModel/AirPodsBatteryViewModel.swift [hunk 1]
```diff
@@ -142,6 +142,7 @@ class AirPodsBatteryViewModel: BluetoothAirpodsBatteryManagementProtocol {
                
                 completion(true, .connected, deviceType)
             case .failure( _):
+                self?.resetAllDevicesBatteryState()
```
Comment (mohamed-arradi) [inline AirpodsPro Battery/AirpodsPro Battery/ViewModel/AirPodsBatteryViewModel.swift:145]: First of all thank you for this PR !

I would prefer that we check first the connection state before resetting. Indeed this method is called quite often and therefore a script running issue can happen but it does not mean that the AirPods are not connected.

I would think on something like that: 

`
let connectionState = self?.connectionStatus ?? .disconnected`

`if connectionState == .disconnected {
self?.resetAllDevicesBatteryState()
}`

What do you think ?

File: .gitignore [hunk 2]
```diff
@@ -7,3 +7,6 @@ AirpodsPro Battery/Pods/Pods.xcodeproj/xcuserdata/mohamed.xcuserdatad/xcschemes/
 AirpodsPro Battery/.DS_Store
 .DS_Store
 AirpodsPro Battery/.DS_Store
+AirpodsPro Battery/AirpodsPro Battery.xcodeproj/xcuserdata/jay.xcuserdatad/xcschemes/xcschememanagement.plist
+AirpodsPro Battery/Pods/Pods.xcodeproj/xcuserdata/jay.xcuserdatad/xcschemes/xcschememanagement.plist
+AirpodsPro Battery/.DS_Store
```
Comment (mohamed-arradi) [inline .gitignore:12]: Can we discard those file. This is related to your own environment and not project related. Those files can be ignore in your local gitignore. 

We can also put `*.xcuserdatad/` instead. 

But for now please discard those changes. I will take care of this one.

File: AirpodsPro Battery/AirpodsPro Battery/ViewModel/AirPodsBatteryViewModel.swift (hunk 1 above)
Comment (makeSmartio) [reply AirpodsPro Battery/AirpodsPro Battery/ViewModel/AirPodsBatteryViewModel.swift:145]: Good point, though this has been working for me - certainly a good idea to double check!

Comment (mohamed-arradi) [reply AirpodsPro Battery/AirpodsPro Battery/ViewModel/AirPodsBatteryViewModel.swift:145]: @makeSmartio no need to close the PR. just update it :)

**Comment by makeSmartio** [comment]
After testing, I stand by my code. It works better and does not fire that frequently.

**Pull request closed without merging**